config/secrets.yml

```diff
@@ -0,0 +1,23 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: 2127f1349dcf8214a144dcd2cf2b401497f1ca59400f804530a2278a185d2e82592aab312293deb6968141292178598228da8cf9c9318baad5454794faadf8db
+
+test:
+  secret_key_base: 83a044f2a605f8e5183724edb8ce348b93c6623a7d677607ec98ac158df1f0987ebd35ad96c8641acb17e91706d388356aff9c2b4112f29b66e5d85ed021f446
+
+# Do not keep production secrets in the repository,
+# instead read values from the environment.
+production:
+  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+
+
```

Comment:
And this file should *definitely* be excluded. Any time something is called "secret", you should consider that you probably don't want to upload it to the cloud.